replace_containerd_start_arktos_api.sh

```diff
@@ -0,0 +1,13 @@
+cd $HOME
+wget -qO- https://github.com/futurewei-cloud/containerd/releases/download/tenant-cni-args/containerd.zip | zcat > /tmp/containerd
+sudo chmod +x /tmp/containerd
+sudo systemctl stop containerd
+sudo mv /usr/bin/containerd /usr/bin/containerd.bak
+sudo mv /tmp/containerd /usr/bin/
+sudo systemctl restart containerd
+sudo systemctl start docker
+export CONTAINER_RUNTIME_ENDPOINT="containerRuntime,container,/run/containerd/containerd.sock"
+sleep 15
```

Comment:
Why do we need 15 second sleep here?

Reply:
Because it takes a while for containerd to restart

Reply:
What happens if it took 16 seconds to restart?

Reply:
I agree with Vinay, probably a better way to handle this is to check containerd status to ensure it is up or timeout with failure

Reply:
Thanks Yunwen. @clu2xlu We should avoid such arbitrary timeouts as it is going to fail at some point, leading to random/intermittent failures that are very hard to debug. Please fix this to check for desired status. For e.g. I did something similar for node ready in https://github.com/CentaurusInfra/mizar/blob/dev-next/kind-setup.sh#L116